### DISCUSSION TITLE
Timing window proportional hit error bar

### DISCUSSION THREAD

**Discussion by Royce551** [ppy/osu#27776] [Ideas]
At the moment, the hit error bar is always the same width or height, regardless of the beatmap's Accuracy. It would be great if there was an option for the hit error bar to scale with the hit windows of the map, so that you can get a feel for how tight the hit windows are. This is how it already works in stable.

**Comment by wwawaawwa** [comment]
this topic was very briefly (and inadequately IMO) discussed in #23748

I would love to see dynamic sizing. Especially for newer players, simply slapping a number next to Accuracy/OD is an incredibly vague way to convey hit window leniency.

Tangentially related but it would also be nice to also have the hit window in ms show on hover like it does in stable.
